src/pages/overview/digital-seal-api/index.md

```diff
@@ -0,0 +1,102 @@
+# Overview
+## What is Digital Seal?
+Digital Seal on a PDF means to digitally sign a PDF document. Digital Seal, alias Digital Signature is a specific type 
+of signature that is backed by a digital certificate, providing proof of your identity. Digital signatures are recognized 
+as being more secure because they’re cryptographically bound to the signed document and can be verified. 
+When you use a digital certificate obtained from a trusted third party, the resulting digital signature is 
+virtually impossible to spoof. It also provides powerful evidence of signer identity, that the signed document was not 
+altered, and that the signatures are valid.
+
+![Digital Seal Identification](../images/blueBar.png)
+
+<!--- 
+https://www.adobe.com/sign/digital-signatures.html
+-->
+
+## Digital Seal Flow
+
+![Digital Seal Process](../images/sealProcess.png)
+
+## Components of Digital Seal
+
+**PDF Document**
+
+Digital Seal API requires a PDF document as an input file on which a seal has to be applied.
+
+**Sealing Parameters**
+
+Digital Seal API requires a few input parameters for the creation of a seal. The parameters should specify the type, 
+format, visibility and appearance of the seal. It should also have the certificate credentials required to communicate with Trust Service Providers.<br/>
+
+Refer [sealing parameters](/overview/digital-seal-api/quickstarts/#2-configure-sealing-parameters) for more details.
+
+**Digital Seal API**
+
+Digital Seal API enables the generation of digital seal on PDF documents with no user interaction. It uses the provided 
+sealing parameters to authorize the client and apply seal on the input PDF document. The generated PDF document is said
+to be bound to the specified legal entity.
+ 
+To Get Started, visit [Quickstarts](./quickstarts.md)
+
+**Signature Image**
+
+Digital Seal API can accept the signature image as input file. This is an optional field.
+
+**Logo Image**
+
+Digital Seal API can accept the logo image as input file. This is an optional field that defaults to 'Acribat Trefoil'
```

Comment:
Should that be Acrobat?

Reply:
Yes, This should be Acrobat. Will rectify this once we review the content as the next step.
Thank You.